tools/istio-clean-iptables/pkg/cmd/cleanup.go

```diff
@@ -0,0 +1,48 @@
+package cmd
+
+import (
+	"istio.io/istio/tools/istio-iptables/pkg/constants"
+	dep "istio.io/istio/tools/istio-iptables/pkg/dependencies"
+)
+
+func removeOldChains(ext dep.Dependencies, cmd string) {
+	// Remove the old chains
+	ext.RunQuietlyAndIgnore(cmd, "-t", constants.NAT, "-D", constants.PREROUTING, "-p", constants.TCP, "-j", constants.ISTIOINBOUND)
+	ext.RunQuietlyAndIgnore(cmd, "-t", constants.MANGLE, "-D", constants.PREROUTING, "-p", constants.TCP, "-j", constants.ISTIOINBOUND)
+	ext.RunQuietlyAndIgnore(cmd, "-t", constants.NAT, "-D", constants.OUTPUT, "-p", constants.TCP, "-j", constants.ISTIOOUTPUT)
+	// Flush and delete the istio chains.
+	ext.RunQuietlyAndIgnore(cmd, "-t", constants.NAT, "-F", constants.ISTIOOUTPUT)
```

Comment:
While you're touching this code, it would be cleaner to iterate over a list of chain names.

Reply:
Done.